### ISSUE TITLE
The performance for 30-shot on COCO 

### ISSUE THREAD

**Issue by shanguangcv** [GuangxingHan/Meta-Faster-R-CNN#8]
Hi, I try to reproduce your method on COCO. I use 8 gpu and keep other default parameters. I can achieve similar performance before fine-tuning. After fine-tuning, I achieve 12.2 AP on 10-shot and 15.2 AP on 30-shot. I think the performance on 30-shot is far lower than your reported performance (16.6 AP). Is there anything in the config file that needs to be changed? Thanks.

**Comment by GuangxingHan** [comment]
Thanks much for your interest in our project and the follow-up experiments!

What is the batch size in each gpu? We train our model using 4 gpus, and the batch size is 8 totally and 2 for each gpu. Can you reproduce the experiments using 4 gpus or increasing the number of batch size? Thanks.

**Comment by shanguangcv** [comment]
I don't change other parameters, so the batch size is still 8 and 1 for each gpu. Using 4 gpus will exceed our memory, so we don't try to use 4 gpus.

**Comment by GuangxingHan** [comment]
In this case, I think you might need to change the learning rate. How about decreasing the learning rate to half, and using longer training iterations?

**Comment by GuangxingHan** [comment]
We have uploaded our meta pre-trained models [here](https://github.com/GuangxingHan/Meta-Faster-R-CNN#pre-trained-models), and confirm that 10/30-shot fine-tuning can reproduce the same results as in the paper, using our provided scripts.
